website/static/js/koHelpers.js

```diff
@@ -459,6 +459,50 @@ ko.bindingHandlers.datePicker = {
 };
 
  /**
+ * Bind content of contenteditable to observable. Looks for maxlength attr
+ * and underMaxLength binding to limit input.
+ * Example:
+ * <div contenteditable="true" data-bind="editableHTML: <observable_name>" maxlength="500"></div>
+ */
+ko.bindingHandlers.editableHTML = {
+    init: function(element, valueAccessor, allBindings, bindingContext) {
+        var $element = $(element);
+        var initialValue = ko.utils.unwrapObservable(valueAccessor());
```

Comment:
Minor: You shouldn't invoke `valueAccessor` if you're going to use `unwrapObservable`. The function is meant to handle that (https://github.com/knockout/knockout/blob/master/src/utils.js#L418).